src/luadebug/symbolize/symbolize_win32.inl

```diff
@@ -75,7 +75,8 @@ namespace luadebug {
     }
 
     inline HANDLE createSymHandler(bool SymBuildPath, bool SymUseSymSrv) {
-        HANDLE proc = GetCurrentProcess();
+        // magic proc
+        HANDLE proc = (HANDLE)788558665;
```

Comment:
这个是什么

Reply:
这个handle应该用magic来替代,不然会出现和其他模块同时使用GetCurrentProcess()当handler的问题.

A handle that identifies the caller. This value should be unique and nonzero, but need not be a process handle. However, if you do use a process handle, be sure to use the correct handle. If the application is a debugger, use the process handle for the process being debugged. Do not use the handle returned by [GetCurrentProcess](https://learn.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess). The handle used must be unique to avoid sharing a session with another component, and using [GetCurrentProcess](https://learn.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess) can have unexpected results when multiple components are attempting to use dbghelp to inspect the current process. Using [GetCurrentProcess](https://learn.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess) when debugging another process will also cause functions like [SymLoadModuleEx](https://learn.microsoft.com/en-us/windows/desktop/api/dbghelp/nf-dbghelp-symloadmoduleex) to have unexpected results.

Reply:
多线程调用Dbghelp的API是被禁止的,不同handle能保证没有这个问题

Reply:
你这么改之后，就找不到符号了。
另外使用magic number，应该加上注释，为什么要这么做。

Reply:
怪我没仔细看,他要求的是必须是一个有效的进程句柄,但是不一定是自己的进程句柄,使用magic会导致可能错误,我还是还原了,这个问题以后在解决吧,没怎么简单